cherab/core/laser/models/laserspectrum_base.pxd

```diff
@@ -0,0 +1,24 @@
+
+from raysect.core.math.function cimport Function1D
+
+from cherab.core.laser.node cimport Laser
+from cherab.core.utility.constants cimport SPEED_OF_LIGHT, PLANCK_CONSTANT
+from numpy cimport ndarray
+
+
+cdef double CONSTANT_HC = SPEED_OF_LIGHT * PLANCK_CONSTANT
+
+cdef class LaserSpectrum_base(Function1D):
```

Comment:
Not PEP8 compliant class name LaserSpectrumBase would be better.... but why not just LaserSpectrum?

Reply:
Renamed to LaserSpectrum